libsrc4/nc4var.c

```diff
@@ -1463,3 +1466,169 @@ nc4_get_typelen_mem(NC_FILE_INFO_T *h5, nc_type xtype, size_t *len)
 
     return NC_NOERR;
 }
+
+
+/**
+ * @internal Check a set of chunksizes to see if they specify a chunk
```

Comment:
Are you moving the chunksize code because chunks now occur in multiple dispatch layers (HDF5 and zarr)?